app/assets/stylesheets/layout/_top_menu.sass

```diff
@@ -133,6 +133,11 @@
           background: $header-item-bg-hover-color
           color: $header-drop-down-item-font-hover-color
 
+    > ul:not(.projects-menu)
+      padding: 8px 0
```

Comment:
Why is it 8px? shouldn't it be 6 like for the first-child in the projects menu?
And can we not reuse that first-child rule so that we do not end up with so many menu specific rules?